src/main/java/org/distributed/systems/ChordStart.java

```diff
@@ -2,18 +2,19 @@
 
 import akka.actor.ActorSystem;
 import akka.actor.Props;
-import org.distributed.systems.chord.actors.Node;
+import org.distributed.systems.chord.actors.NodeActor;
 
 public class ChordStart {
 
     public static final int STANDARD_TIME_OUT = 10000;
     // FIXME 160 according to sha-1 but this is the max_length of a java long..
-    public static final int M = 64;
+    public static final int M = 3; // Number of bits in key nodeId's
```

Comment:
Is this also working for the later version, that we need to benchmark? I would assume M is larger in that case?

Maybe use a ENV and config var

Reply:
Yeah something like env var would be nice. 👍

Reply:
Okay, guess let's add that to Trello